lib/model/folder_summary.go

```diff
@@ -330,9 +383,9 @@ func (c *folderSummaryService) sendSummary(ctx context.Context, folder string) {
 	if err != nil {
 		return
 	}
-	c.evLogger.Log(events.FolderSummary, map[string]interface{}{
-		"folder":  folder,
-		"summary": data,
+	c.evLogger.Log(events.FolderSummary, folderSummaryEventData{
```

Comment:
Double check that verbose service doesn't crash due to this change as well?

Reply:
Right, of course it crashes. Checked all usages of `events.FolderSummary` now and we should be good. (Somehow whenever a change comes up as a followup, I tend to skip the "think about consequences" step of doing changes...).